public/app/core/components/Login/LoginLayout.tsx

```diff
@@ -45,7 +45,7 @@ export const LoginLayout = ({ children, branding }: React.PropsWithChildren<Logi
         </div>
         <div className={loginStyles.loginOuterBox}>{children}</div>
       </div>
-      <Footer customLinks={branding?.footerLinks} />
+      {branding?.hideFooter ? <></> : <Footer customLinks={branding?.footerLinks} />}
```

Comment:
I think it's better to return `null` for no content, however this code can be simplified even more: `{!branding?.hideFooter && <Footer customLinks={branding?.footerLinks} />}`

Reply:
I agree